router/server.go

```diff
@@ -92,8 +93,8 @@ func main() {
 
 	proxyProtocol := os.Getenv("PROXY_PROTOCOL") == "true"
 
-	httpPort := flag.Int("http-port", 8080, "http listen port")
-	httpsPort := flag.Int("https-port", 4433, "https listen port")
+	httpPortsRaw := flag.String("http-port", "8080", "http listen port(s)")
+	httpsPortsRaw := flag.String("https-port", "4433", "https listen port(s)")
```

Comment:
I think this configuration should probably be moved to an environment variable so that it can be configured via `flynn env set`